docs/source/contributing.rst

```diff
@@ -34,30 +34,46 @@ Development
 
 Wagtailtrans made it very easy to setup a runnable Django project to help with the development. It ships with a Sandbox application that can be availed for this purpose. You need to have some additional packages installed and a PostgreSQL Database on your local machine.
 
-* We suggest you use a virtual environment for development.
+* Clone a copy of the `Wagtailtrans codebase <https://github.com/LUKKIEN/wagtailtrans>`_
 
-* Activate your virtual environment and install following packages.
+  .. code-block:: bash
 
-.. code-block:: bash
+    $ git clone https://github.com/LUKKIEN/wagtailtrans.git
+    $ cd wagtailtrans.
 
-    Django
-    Wagtail
-    psycopg2
-    pytest
-    pytest-cov
-    pytest-django
-    coverage
-    factory-boy
+* **Setup Database**
 
-* Set PostgreSQL Database Authentication parameters host, user, and password as environment variables.
+  There is a couple of different ways to do this. First, you can place your local settings in ``tests/_sandbox/settings/local_settings.py``. You can provide a standard `django database settings <https://docs.djangoproject.com/en/1.10/ref/settings/#databases>`_ here to set your database up. (A sample copy is provided in  ``tests/_sandbox/settings/local_settings.sample``.)
```

Comment:
Change: There is a couple of different ways to do this -- ``There are a couple of ways to do this``